modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -252,6 +217,9 @@ else if (cfg.getCacheMode() == REPLICATED) {
                     if (aff.getHashIdResolver() == null)
                         aff.setHashIdResolver(new AffinityNodeAddressHashResolver());
                 }
+            } else if (cfg.getCacheMode() == LOCAL && !(cfg.getAffinity() instanceof LocalAffinityFunction)){
+                cfg.setAffinity(new LocalAffinityFunction());
+                U.warn(log, "Only LoalAffinityFunction() is applicable for local cache [cacheName=" + U.maskName(cfg.getName()) + ']');
```

Comment:
Please set text message to the one that starts this way

AffinityFunction configuration parameter will be ignored for local cache [cacheName...